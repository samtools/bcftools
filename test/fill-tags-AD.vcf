##fileformat=VCFv4.2
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depth">
##contig=<ID=chr1,length=249250621,assembly=hg19>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B	C
chr1	10153	.	AC	A	.	.	.	AD	10,1	20,2	30,3
chr1	10153	.	AC	A	.	.	.	AD	10,1	.	.
