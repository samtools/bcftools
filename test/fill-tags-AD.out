##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depth">
##contig=<ID=chr1,length=249250621,assembly=hg19>
##INFO=<ID=DP,Number=1,Type=Integer,Description="Added by +fill-tags expression DP:1=int(sum(AD))">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B	C
chr1	10153	.	AC	A	.	.	DP=66	AD	10,1	20,2	30,3
chr1	10153	.	AC	A	.	.	DP=11	AD	10,1	.	.
